docs/podman-replay-kube.1.md

```diff
@@ -0,0 +1,70 @@
+% podman-generate Podman Man Pages
+% Brent Baude
+% December 2018
+# NAME
+podman-replay-kube - Recreate pods and containers based on Kubernetes YAML
+
+# SYNOPSIS
+**podman replay kube **
+[**-h**|**--help**]
+kubernetes_input.yml
+
+# DESCRIPTION
+**podman replay kube** will read in a structured file of Kubernetes YAML.  It will then recreate
+the pod and containers described in the YAML.  The containers within the pod are then started and
+the ID of the new Pod is output.
```

Comment:
Can only one pod be created or possibly multiples based on the yaml?
"new Pod" to "new pod".

Reply:
Only one at a time for now.